src/components/DocumentSelector/index.js

```diff
@@ -103,6 +108,7 @@ const withDefaultOptions = (types: Object, group: GroupType) =>
           }
         })
       }
+      }
```

Comment:
Somethings' wrong with indentation here as you have two `}` coming one after another in the same ident.

Reply:
Yes, indeed. Will fix that.